Post: Extending material theme in Jetpack Compose. (#146)
xmartlabs/blog: _posts/2021-05-10-compose-themes.md

    textSizeRegular = 16.sp,

    listItemVerticalPadding = 27.dp,
    listItemHorizontalPadding = 30.dp,
)

private val smallDeviceAppDims = AppDims(
    textSizeSmall = 12.sp,
    textSizeMedium = 13.sp,
    textSizeRegular = 14.sp,

    listItemVerticalPadding = 15.dp,
    listItemHorizontalPadding = 14.dp,
)
```

Then, you have to add the `AppDims` to your `AppTheme` just like we did with the `AppColors` example.
Check out [the complete code in Gong repo](https://github.com/xmartlabs/gong/blob/b0b617e/app/src/main/java/com/xmartlabs/gong/ui/theme/AppDims.kt).

# Conclusion
I don't have to convince you about the importance of having a good UX/UI in an app, most of the time we use top-tier apps that provide a world-class experience and we end up expecting this UI/UX quality everywhere.

Compose is coming to transform the way we build Android apps, it's a whole new approach that drastically simplifies and accelerates UI development on Android.

Theming the application using Material Theme helps when it comes to encapsulating and reutilizing styles and assets.
There is nothing worse than having a huge app that is a mess regarding theme and assets management and configuration.

> In this blog post, we present concepts and ideas to leverage compose, app theming, and Material and if your app still needs more customization, be able to extend material capabilities.

Reviewer: I think it's clearer this way:

```suggestion
This blog post presents concepts and ideas to leverage compose, app theming, and Material. If your app still needs more customization, you'll be able to extend Material capabilities.
```

Author: Maybe?
```suggestion
This blog post presents concepts and ideas to leverage compose, app theming, and Material.
If your app still needs more customization, you'll be able to extend Material capabilities to derive full benefit from them.
```
